HTML/Mechanus Cipher.html

```diff
@@ -0,0 +1,76 @@
+﻿<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+        <title>Mechanus Cipher — Keep Talking and Nobody Explodes Module</title>
+        <meta name="viewport" content="initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="css/main.css">
+        <link rel="stylesheet" type="text/css" href="css/font.css">
+        <script src="js/ktane-utils.js"></script>
+        <style>
+            table {
+                margin: 1em auto;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="section">
+            <div class="page page-bg-01">
+                <div class="page-header">
+                    <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                    <span class="page-header-section-title">Mechanus Cipher</span>
+                </div>
+                <div class="page-content">
+                    <img src="img/Component/Mechanus Cipher.svg" class="diagram">
+                    <h2>On the Subject of Mechanus Cipher</h2>
+					<p class="flavour-text">A cipher named after the plane of order, where a creature lurks to bring structure.</p>
+
+					<p>The module displays two rows of lettered panels, an extra lettered panel, a submit panel, and two lettered gears.</p>
+					<p>You can press any of the panels on a row to flip that row. The top row shows an encrypted word, a keyword and a binary string. The bottom row shows two sets of numbers and a second keyword. The extra panel shows a key letter.</p>
+					<p>To solve the module, decrypt the word following the steps below and submit it using the gears. To submit, press the gears to turn them, and press sub when you want to enter the selected letter. You can flip the panels to clear your input.</p>
+					<h3>Decrypting the word</h3>
+					<h4>Step 0 - Creating your key</h4>
+					<p>For steps 2 and 3 you need a key. To make this key, take your keyword from the top row and repeat it until you have 26 letters. Then, for all letters in this string in reading order, if a letter has already occurred, shift it forward in the alphabet until this is not the case.</p>
+					<p style="font-size: small"><strong>Example:</strong> TROUPER -> TROUPERTROUPERTROUPERTROUP -> TROUPESVWQXYFZABCDGHIJKLMN</p>
```

Comment:
Please use html escape characters for > and < that is not a part of html tags. Use `&gt;` for > and `&lt;` for <